jscomp/stdlib/camlinternalOO.cppo.ml

```diff
@@ -96,26 +96,14 @@ let public_method_label s : tag =
 (**** Sparse array ****)
 
 module Vars =
-#ifdef BS
-  Belt.Map.String
```

Comment:
Users of the standard library will lose this "optimization" to not use a functor, do we want to inline Belt.Map.String perhaps?

Reply:
This is a very good question, that I hope I can answer sufficiently:

- I think short term the answer is "we don't care" because the benefits (removing belt as a dep) justify it, and the code becomes the same as the upstream stdlib
    - even if it did, I don't think the impact would be important -- and Melange users barely use OCaml objects anyway.
- longer term we want to add continuous benchmarks to melange, which we've added to the next quarter roadmap. Hopefully we can measure the impact of this change retroactively then.

Reply:
> and Melange users barely use OCaml objects anyway.

Not sure I get this part, could you elaborate? How are objects related to the discussion about larger bundles when using functors?

Reply:
Oh, I see, this is the `camlinternalOO` module 🤦